client/src/telemetry/metrics.yaml

```diff
@@ -37,6 +37,23 @@ page:
     notification_emails:
       - mdn-team@mozilla.com
     expires: 2023-09-05
+  http_status:
+    type: string
```

Comment:
Could this not be a numeric type?

Reply:
Yes, I debated this a little bit: in glean the "numeric" type is "quantity" and requires a unit, but a http status code is unitless! Due to that, I reckoned a string made more sense.